client/app/assets/less/ant.less

```diff
@@ -47,6 +47,12 @@
   z-index: 1050;
 }
 
+// Button overrides
+
+.@{btn-prefix-cls} {
+  transition: none;
```

Comment:
What's this for?

Also, might be redundant since this rule overrides it:
```css
.ant-table * {
    transition: none!important;
}
```

Reply:
Yeah the transition is a bit slow.
I propose this instead:
```css
transition-duration: 150ms;
```

Reply:
@ranbena  80-90ms feels more comfortable imo. wdyt?

Reply:
For me 100 was too fast, 200ms too slow, 150 sweet spot

Reply:
"too fast" as in meaningless

Reply:
Done. With `80ms` animation is still noticeable, but not annoying

Reply:
100ms feels to slow for me :smile:

Reply:
@ranbena  Tried once more, and seems you're right - 150ms is better :thinking: Changed it again

Reply:
![perfection1](https://user-images.githubusercontent.com/486954/52488291-2a05fd80-2bc8-11e9-933f-01b9cec12c0d.jpg)

Reply:
Is it possible there is a difference between MacOS and Linux?

I remember feeling Ant's animation are slow, while for Levko they were fine. Maybe something similar happens here?

Reply:
I think it's just different perception

Reply:
There might be something to @arikfr's idea though. I had a similar though about some animations that @ranbena used in his new dashboard parameters UI. Might be interesting to double check.